src/sentry/api/endpoints/group_events.py

```diff
@@ -85,7 +85,11 @@ def _get_events_snuba(
             "start": start if start else default_start,
             "end": end if end else default_end,
         }
-        direct_hit_resp = get_direct_hit_response(request, query, params, "api.group-events", group)
+        referrer = f"{group.issue_category}.api.group-events"
```

Comment:
Please add the new referrer's here https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/referrer.py#L43